README.md

```diff
@@ -1,139 +1,169 @@
 # player-loader-webpack-plugin
+The official webpack plugin for the Brightcove Player.
 
 ## Installation
-To install just use
+To install, use:
 
 ```sh
-npm i -D @brightcove/player-loader-webpack-plugin
+npm install --save-dev @brightcove/player-loader-webpack-plugin
 ```
 
-
-## Using the plugin
-
-First require the plugin at the top of your `webpack.config.js`:
+## Basic Usage
+First, require the plugin at the top of your `webpack.config.js`:
 
 ```js
 const PlayerLoader = require('@brightcove/player-loader-webpack-plugin');
 ```
 
-Then create an instance of the `PlayerLoader` plugin in the `plugins` array of your `webpack.config.js`:
+Then create an instance of the `PlayerLoader` plugin in the `plugins` array:
 
-> Note: That `accountId` is a required option!
+> **Note:** `accountId` is a required parameter!
 
 ```js
-  plugins: [
-    new PlayerLoader({accountId: '12345678910'})
-  ],
+plugins: [
+  new PlayerLoader({accountId: '12345678910'})
+]
 ```
 
-For more information on options see the options section of the README.
+For a full list of options, see the [Options section](#options) below.
 
-## Options
+## How it Works
+This webpack plugin will prepend your Brightcove Player to your bundle. Doing this can reduce the number of requests needed by your website and ensure that the global `bc` function is available synchronously.
 
-### `accountId`
-* **REQUIRED**
-* *Type:* `string` | `number`
+### Limitations
+There are several limitations and caveats to using this plugin.
 
-A Video Cloud account ID.
+1. iframe embeds are not supported. This plugin is only suitable for use with in-page/advanced embeds.
+1. If your player is updated or republished, the bundle will need to be re-generated before the bundled player is updated.
 
-### `embedId`
-* *Type:* `string`
-* *Default:* `'default'`
+As an alternative, users who want an iframe embed or want to load their player script asynchronously can use the [@brightcove/player-loader](https://github.com/brightcove/player-loader) project instead, which downloads players at runtime rather than at build time. It does not have these limitations.
 
-The Brightcove Player [embed ID](https://support.brightcove.com/guide-embed-apis) for the player. The default value is correct for most users.
+### Auto Setup
+When your bundle executes, the Player will automatically set up any embed elements that match certain criteria:
 
-### `playerId`
-* *Type:* `string`
-* *Default:* `'default'`
+1. Must be either a `<video>` or `<video-js>` element.
+1. Must have a `data-player` attribute that is equivalent to the `playerId` of the bundled player.
```

Comment:
I believe it has to have an account id as well

Reply:
It doesn't. Player ID and embed ID. Just checked.

Reply:
Huh, guess so.